content/en/docs/instrumentation/js/manual.md

```diff
@@ -228,6 +452,14 @@ app's execution could continue.
 
 In most cases, stick with `BatchSpanProcessor` over `SimpleSpanProcessor`.
 
+#### Configure an exporter
```

Comment:
Reading this page, I think we could do well to have a code sample.

First, elaborate that the default exporter is `localhost:4317`, which means that if they also configure a collector (link to this...somehow), then it will send to that collector by default.

Then, explicitly say that you're wiring up the console exporter for the purposes of local development/seeing the instrumentation easily. And also include a comment in the source code that says it's for local development and debugging only.

Reply:
> First, elaborate that the default exporter is localhost:4317, which means that if they also configure a collector (link to this...somehow), then it will send to that collector by default.

I tried to verify this, but it's actually not the case, here's my instrumentation.ts:

```js
/*instrumentation.ts*/
import { BatchSpanProcessor, ConsoleSpanExporter } from "@opentelemetry/sdk-trace-base";
import { Resource } from "@opentelemetry/resources";
import { SemanticResourceAttributes } from "@opentelemetry/semantic-conventions";
import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node";

const resource =
  Resource.default().merge(
    new Resource({
      [SemanticResourceAttributes.SERVICE_NAME]: "dice-server",
      [SemanticResourceAttributes.SERVICE_VERSION]: "0.1.0",
    })
  );

const provider = new NodeTracerProvider({
    resource: resource,
});
// const exporter = new ConsoleSpanExporter();
// const processor = new BatchSpanProcessor(exporter);
// provider.addSpanProcessor(processor);

provider.register();
```

I also ran it with debugging on, but I don't see any traces being exported. I know that this is true if you use the NodeSDK setup steps (`const sdk = new NodeSDK({ ... }); sdk.start()`).

Reply:
Wondering if the setup section should work with NodeSDK and only comment on the capability to do low-level wiring yourself 🤔 

cc @open-telemetry/javascript-approvers

Reply:
Hrm, I guess I was thinking in terms of the NodeSDK. I think we should finally switch to the Node SDK for this page as well. It was less stable when I initially did this section, but it's been out long enough that we should just do the switch.

Reply:
Sorry for the late response: As much as it pains me to say, I'm always hesitant to recommend the `NodeSDK` as it has grown over the years and was never fully thought through. It needs a refactor/rewrite, is missing many features one would expect, and should not be considered stable (as indicated by the `0.x` version). It also pulls in a lot of dependencies that cause trouble in certain enviornments (such as the jaeger exporter). 

Therefore, until we address those issues, I think making people depend on its behavior may be doing more harm than good in the long term.

Reply:
thanks @pichlermarc. I was not aware of this status of `NodeSDK`. We also use it through out the getting started docs for JavaScript and a few other places (can collect them as needed). So this sounds we should keep things as they are with the non-NodeSDK-approach?

Reply:
How likely are proposed changes to be a big deal? In practice the NodeSDK has been pretty stable, so much so that we (hoenycomb) have been able to rely on it for almost a year in our own distribution. There was a breaking change at one point but it was really simple to update.

Reply:
I guess this might be worth a SIG convo, but I personally think that's fine enough to document. I would wager the large majority of usage is just in passing in config options and running `start`/`shutdown`.

Reply:
I'll bring it up at this week's JS SIG 🙂 - I think as long as only constructor/start and shutdown are used in the docs this should be fine. 

If the others in the JS SIG meeting on Wednesday agree, I'll set steps to mark these methods as deprecated, then remove them in a later release.

Reply:
Sounds good! I would also say that if there's some known "this will break people" work that you're planning on, if that work being done helps get us documenting the NodeSDK, then I'd support frontloading it.

Reply:
We discussed it in the SIG. There were no objections to removing the said methods, so I opened a PR to deprecate them. If it goes through we will remove them in later releases. https://github.com/open-telemetry/opentelemetry-js/pull/3996

The config options of NodeSDK will remain as-is (some logs support will be added in the upcoming release).

Long story short: It's safe to move on with using `@opentelemetry/sdk-node` in the docs :slightly_smiling_face:

Reply:
That's superb, thanks @pichlermarc!